### PR TITLE
Add com.wiz.Note

### PR DESCRIPTION
**Pull request by cheese**
Based on https://github.com/flathub/flathub/pull/910

### Changes since then
- Update to 2.8.5
- Convert to YAML manifest
- Adapt to Qt 5.15, use io.qt.qtwebengine.BaseApp
- Add `--talk-name`'s for systray icon
- Remove `--persist=.local`
- Set TMPDIR to allow single instance

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/WizTeam/WizQTClient/issues/587
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by barthalion** [comment]
bot, build com.wiz.Note

**Comment by barthalion** [comment]
```
builddir/files/share/appdata/com.wiz.Note.appdata.xml: FAILED:
• url-not-found         : <screenshot> failed to download (HTTP 404: Not Found) [http://www.wiz.cn/www/img/index/team.png]
```

Screenshots are no longer available.

File: com.wiz.Note.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.wiz.Note</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0+</project_license>
+  <name>Wiz Note</name>
+  <summary>An excellent PKM(personal knowledge management) desktop environment based on cloud usage</summary>
```
Comment (AsciiWolf) [inline com.wiz.Note.appdata.xml:7]: Typo: Missing space before the bracket.

File: com.wiz.Note.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  <description>
+    <p>
+      The project is based on Qt, aimed to provide an excellent PKM(personal knowledge management) desktop environment based on cloud usage.
```
Comment (AsciiWolf) [inline com.wiz.Note.appdata.xml:10]: Same typo as above.

File: com.wiz.Note.appdata.xml (hunk 1 above)
Comment (cheese) [reply com.wiz.Note.appdata.xml]: Fixed

File: com.wiz.Note.appdata.xml (hunk 2 above)
Comment (cheese) [reply com.wiz.Note.appdata.xml]: Fixed

**Comment by cheese** [comment]
bot, build com.wiz.Note

**Comment by barthalion** [comment]
http://cdn.wiz.cn/wp-content/uploads/2014/12/NightMode-1024x608.png keeps timeouting on our infra. It would be better to just add them to the git repo and use direct URL to GitHub.

**Comment by cheese** [comment]
bot, build com.wiz.Note

**Comment by cheese** [comment]
bot, build com.wiz.Note

**Pull request closed without merging**